editor/components/meta-boxes/meta-boxes-area/index.js

```diff
@@ -89,9 +90,7 @@ class MetaBoxesArea extends Component {
 	}
 
 	getFormData() {
-		const data = new window.FormData( this.form );
-		const entries = Array.from( data.entries() );
-		return entries;
+		return jQuery( this.form ).serialize();
```

Comment:
It returns a string in here after changes got applied.

`getFormData` is compared against `this.originalFormData` which is initialized as an empty array.
```
this.originalFormData = [];
```

Reply:
Good catch, changing the initial value :)